modules/ROOT/pages/database-administration/composite-databases/manage-composite-databases.adoc

```diff
@@ -0,0 +1,170 @@
+[role=enterprise-edition not-on-aura]
+[[composite-databases]]
+= Managing composite databases
```

Comment:
We say something very similar currently in the new page: composite-databases-concepts

Reply:
don't think I've gotten to that one yet (or if I did I didn't look specifically for this hence my note to confirm it later on, since I'm reviewing in chunks it felt like the easiest way of keeping track of things without pausing the current file's review to go look it up 🤷)

Reply:
(unresolved only cause it's on my list of things to double check, will resolve once I've actually double checked it)

Reply:
It's sufficiently covered 👍